### DISCUSSION TITLE
Disable PSR extension

### DISCUSSION THREAD

**Discussion by YamiKnigth** [Ym0T/pterodactyl-nginx-egg#13] [Q&A]
Im trying to install azuriom using the egg on my panel but i got this error anytime i tried to install it https://pastebin.com/nvEKuDG3

**Comment by Ym0T** [comment]
> Im trying to install azuriom using the egg on my panel but i got this error anytime i tried to install it https://pastebin.com/nvEKuDG3

Hello 👋 It looks like the issue is a compatibility problem with the version of Monolog you're using. The emergency() method expects a different parameter type (Stringable|string) than what's defined in the LoggerInterface, causing a conflict.

Try running composer update to ensure all dependencies are up to date and compatible. Also, make sure your autoloading is set up properly by running composer dump-autoload.

You can deactivate the psr in the php config.

**Comment by YamiKnigth** [comment]
sorry for the question but how can i disable it on the php config?

**Reply by Ym0T** [reply]
The easiest way is to remove the psr extension from your fork and make your own Docker images out of it.
In fact, I can't think of a way to disable that via the config. If you look at the functionality of the egg. sorry.

**Reply by YamiKnigth** [reply]
i dont know how to create docker images, the last version of docker image works fine for me when doesnt have to many dependencies

**Reply by Ym0T** [reply]
What PHP version do you need? I will create a custom egg with a custom dockerfile for you tonight.

**Reply by YamiKnigth** [reply]
i alrready make a docker file without psr and it works thank you so much
